README.md

```diff
@@ -23,7 +23,8 @@ https://github.com/DataBiosphere/data-explorer-indexers, or any other indexer.
   * If you used your own indexer, the config files must follow
```

Comment:
In api.yaml, do you mind adding this description to tableNames:
```
Fully-qualified table name: <project id>.<dataset id>.<table name>
```

Reply:
Done